src/qililab/result/qprogram/qprogram_results.py

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 
 """MeasurementResult class."""
+import numpy as np
+
+from qililab.result.counts import Counts
```

Comment:
Another clue for the reorganization I ask below, is the fact that now QProgram module depends on `Counts` which is outside of QProgram module. We try for QProgram to be completely independent.